src/taskgraph/util/path.py

```diff
@@ -93,13 +93,8 @@ def basedir(path, bases):
 
 
 re_cache = {}
-# Python versions < 3.7 return r'\/' for re.escape('/').
-if re.escape("/") == "/":
-    MATCH_STAR_STAR_RE = re.compile(r"(^|/)\\\*\\\*/")
-    MATCH_STAR_STAR_END_RE = re.compile(r"(^|/)\\\*\\\*$")
-else:
-    MATCH_STAR_STAR_RE = re.compile(r"(^|\\\/)\\\*\\\*\\\/")
-    MATCH_STAR_STAR_END_RE = re.compile(r"(^|\\\/)\\\*\\\*$")
+MATCH_STAR_STAR_RE = re.compile(r"(^|/)\\\*\\\*/")
+MATCH_STAR_STAR_END_RE = re.compile(r"(^|/)\\\*\\\*$")
```

Comment:
Depends on #207 ?

Reply:
Yep, sorry I was planning to rebase before requesting review.